src/Curl/Curl.php

```diff
@@ -124,19 +124,19 @@ class Curl
     public $http_error_message = null;
 
     /**
-     * @var string|array TBD (ensure type) Contains the request header informations
+     * @var string|array TBD (ensure type) Contains the request header information
      */
     public $request_headers = null;
 
     /**
-     * @var string|array TBD (ensure type) Contains the response header informations
+     * @var string|array TBD (ensure type) Contains the response header information
      */
     public $response_headers = array();
 
     /**
-     * @var string Contains the response from the curl request
+     * @var string|false Contains the response from the curl request
      */
-    public $response = null;
+    public $response = false;
```

Comment:
This can break libraries using it.

Reply:
as from php.net/curl docs, when curl_exec fails it returns false. so in this case $response will be false. in this case i think will be better when response is not set to return always false, instead of null|false|string

Reply:
> Returns TRUE on success or FALSE on failure. However, if the CURLOPT_RETURNTRANSFER option is set, it will return the result on success, FALSE on failure.